packages/freetype_sub.rb

```diff
@@ -1,46 +1,13 @@
 require 'package'
 
 class Freetype_sub < Package
-  description 'Freetype_sub is a version without harfbuzz. It is intended to handle circular dependency betwwen freetype and harfbuzz.'
+  description 'Freetype_sub was a version w/o harfbuzz, now handled within harfbuzz'
   homepage 'https://www.freetype.org/'
-  version '2.11.0'
+  version '1.0'
```

Comment:
Follow `freetype` version? (otherwise this may brake `crew update`)

Reply:
This is intentional, to force anyone with freetype_sub installed to be updated to the full freetype version.

Reply:
(Freetype_sub is no longer a dependency for any package.)

Reply:
Oh it should be "break"😅

Reply:
As far as I can tell, `crew update` doesn't do monotonic versioning. It just cares that the version has changed _at all_ from what it was before.